pods-framework/pods
# Improve repeatable UI

## Description


This PR aims to improve the repeatable fields UI.

1. Make it smaller and more integrated in the WP core UI elements.
2. Similar background and hover colors with WP core panels
3. Improve dragging (border handling)

## Related GitHub issue(s)




Fixed #6853 

## Testing instructions



1. Create some repeatable fields
4. See current UI
5. Rebuild DFV with the changes in this PR
6. See new UI :)

## Screenshots / screencast


### Current (2.9.1):
![image](https://user-images.githubusercontent.com/826148/184242214-cde58b9d-29c7-46d3-b622-fdf30710837c.png)
Large fields:
![image](https://user-images.githubusercontent.com/826148/184335257-b71af1dd-d54a-467c-9c7a-9a7f56d14f11.png)

### Suggested:
![image](https://user-images.githubusercontent.com/826148/184242370-4c7390fd-0735-4f04-962c-2dcd8ffb1ed7.png)
Large fields:
![image](https://user-images.githubusercontent.com/826148/184335437-11c0b35c-94e5-41f7-b962-0c78f5d8dda6.png)
Hover & Draggins:
![Edit-Person-“Person-Y”-‹-5-Hurkadurk-—-WordPress](https://user-images.githubusercontent.com/826148/184340346-771d9e96-0d9e-44cd-8935-ca94345e70b2.gif)

## Changelog text for these changes







## PR checklist
